IC-benchmarking-env.yml

```
@@ -0,0 +1,15 @@
+#modeled off of  "NF1_SchwannCell_data_analysis/environment.yml" & "nf1_cellpaining_data/nf1_cellpainting_env.yml"
+name: IC-benchmarking-env
+channels:
+  - conda-forge
+dependencies:
+  - conda-forge::python=3.9
+  - conda-forge::pip
+  - conda-forge::numpy
```

```suggestion
  - conda-forge::numpy
  - conda-forge::pandas
  - conda-forge::ipykernel
  - conda-forge::jupyterlab
```
Recommend including these dependencies if you plan on using this environment to create jupyter notebooks to work with the CellProfiler data later on.